File: docs/archive/check_schema.py
import sqlite3, json, os
db_path = os.path.join('..', 'app.db')
conn = sqlite3.connect(db_path)
cur = conn.cursor()

# Get sheets schema
cur.execute("PRAGMA table_info('sheets')")
print("SHEETS COLUMNS:", [c[1] for c in cur.fetchall()])

# Get columns schema  
cur.execute("PRAGMA table_info('columns')")
print("COLUMNS FIELDS:", [c[1] for c in cur.fetchall()])

# Check for records tables
cur.execute("SELECT name FROM sqlite_master WHERE type='table' AND name LIKE 'records_%'")
print("RECORDS TABLES:", [r[0] for r in cur.fetchall()])

conn.close()

File: docs/archive/check_schema.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('app.db');

db.all("PRAGMA table_info(sheets)", [], (e, rows) => {
  console.log('sheets columns:', JSON.stringify(rows, null, 2));
});

db.all("PRAGMA table_info(columns)", [], (e, rows) => {
  console.log('columns columns:', JSON.stringify(rows, null, 2));
});

db.close();